tmt/utils.py

```diff
@@ -1332,8 +1333,11 @@ def yaml_to_dict(data: Any,
 
 
 def yaml_to_list(data: Any,
-                 yaml_type: Optional[YamlTypType] = 'safe') -> List[Any]:
+                 yaml_type: Optional[YamlTypType] = 'safe') -> List['tmt.result.ResultData']:
```

Comment:
I'd stick to `Any`, this function might be used from other places as well. It is not right now, but since it's in `utils` package, and it's generic enough, this may change. I'd not touch this function.